pandas/_libs/tslibs/resolution.pyx

```diff
@@ -10,7 +10,7 @@ np.import_array()
 
 from util cimport is_string_object, get_nat
 
-from khash cimport (
+from pandas._libs.khash cimport (
```

Comment:
I think you can do relative imports here?

``from ..khash cimport ``? not sure. but this is ok for now.